### PR TITLE
Welcome darlene

### PR DESCRIPTION
**Pull request by COpenda**
@rowo please review and merge.

### CODE REVIEW

**Comment by rowo** [comment]
@COpenda I removed "The month of August comes with yet more surprises. We might just have to move to a much bigger office to accommodate the growing team." I am not sure why it is a surprise and the second line seems like fluff. We can be straightforward in these new employee posts. I'm all for being light-hearted, but the writing should still be efficient.

Also sorry, I removed the *we are going to play one-on-one* part because it was oddly worded because the post isn't written in the first person but then you say "we" and it was confusing until I realized you mean you, the author. If you want to keep it in, maybe it can be worded better.

Also I removed the last paragraph.. seemed like too many extracurriculars at that point. What would be cool is more detail around what interests her about computer interactive statistics or something related to what she'll do at Ona, if that's known.

**Comment by COpenda** [comment]
@rowo that's fine. For the last paragraph, she has no specific project at the moment, she's authoring forms for one of the consulting customers.

**Comment by rowo** [comment]
@jmunene Do you know what Darlene will be helping out on?

**Comment by rowo** [comment]
@COpenda You can still write about what she'll be doing which is:
- Ona.R work
- support
- xlsform authoring
- report authoring
- providing great services for consulting clients like all of the stuff on https://ona.io/services.html

File: _posts/2016-08-08-welcome-darlene.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+type: post
+published: true
+---
+
+![Welcome Darlene](/assets/images/2016-08-08/darlene.jpg)
+
+
+Meet Darlene Kwamboka, who has joined Ona as a data analyst intern.
+
+Darlene is looking forward to graduating with a degree in statistics from Jomo Kenyatta University of Agriculture and Technology this November. Like most of us, she had no idea of what she wanted to do in life prior to starting college. All she knew was she had a love for science and mathematics. This informed her course selection and as it turns out, she enjoyed statistics classes with <i>Computer Interactive Statistics</i> being her favorite. At Ona, Darlene will use her statistics skills to play a major role in [ona.R](https://blog.ona.io/general/2016/04/15/Ona-R-Integration.html). She will also be supporting our [technical consulting services](https://ona.io/services.html) in authoring XLS forms and reports.
```
Comment (pld) [inline _posts/2016-08-08-welcome-darlene.md]: end of para, use XLSForms, that's what the xlsform website uses

**Pull request merged**